app/views/find/v2/results/index.html.erb

```diff
@@ -13,6 +13,11 @@
             <%= form.govuk_check_box :can_sponsor_visa, "true", multiple: false, label: { text: t("helpers.label.courses_query_filters.can_sponsor_visa"), size: "s" }, checked: params[:can_sponsor_visa].present? %>
           <% end %>
 
+          <%= form.govuk_check_boxes_fieldset :study_types, legend: { text: t("helpers.legend.courses_query_filters.study_type_html"), size: "s" }, class: "app-filter__group govuk-checkboxes--small", hidden: false do %>
+            <%= form.govuk_check_box :study_types, "full_time", label: { text: t("helpers.label.courses_query_filters.study_type_options.full_time"), size: "s" }, checked: params[:study_types].present? && params[:study_types].include?("full_time") %>
+            <%= form.govuk_check_box :study_types, "part_time", label: { text: t("helpers.label.courses_query_filters.study_type_options.part_time"), size: "s" }, checked: params[:study_types].present? && params[:study_types].include?("part_time") %>
+          <% end %>
```

Comment:
Are we happy with these checked inteligence? It feels too much to abstract but also feels wrong to be here. But because we don't have a form object this needs to be here. Let me know your thoughts.

Reply:
I would probably put it in the controller and use instance variables. If the messy logic should go anywhere, move it as high up the stack as possible

Reply:
My gut feeling is the `checked: ` key shouldn't be needed. It doesn't do anything out of the ordinary.

Reply:
For context, I did had the search form on an old PR:
https://github.com/DFE-Digital/publish-teacher-training/pull/4751/commits/d1b20e12d7aaa5299ce28cf4fa3c38128e33a144#diff-b05ade9e136163e376907cdd8caee4d0f89733201c361cdc6dcadfc7277d6635R9

This would be move there and the checked won't be needed and all params handling would be there in terms of formating and checked.